root/layout/components/BottomMenu.js

```diff
@@ -9,7 +9,7 @@
 
 import * as React from 'react';
 
-import {CatalystContext} from '../../context.mjs';
+import {CatalystContext, SanitizedCatalystContext} from '../../context.mjs';
```

Comment:
Can we allow `stash.server_languages` in `SanitizedCatalystContext` and always use that? It doesn't seem like something that wouldn't belong and using sanitized always when possible seems like a good plan anyway.